ci/shell.nix

```diff
@@ -23,7 +23,7 @@ let
     ${nixpkgs.patchelf}/bin/patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" "$out"
     chmod -w $out
   '';
-  rustStable = nixpkgs.rust-bin.stable."1.53.0".default.override {
+  rustStable = nixpkgs.rust-bin.stable."1.54.0".minimal.override {
```

Comment:
I am trying to change `default` to `minimal` here (mainly to exclude rust-docs), but I am not sure whether this would work (minimal also does not include e.g. rustfmt and clippy).